web/js/components/util/list.js

```diff
@@ -2,6 +2,42 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { ListGroup, ListGroupItem, Badge } from 'reactstrap';
 
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faArrowCircleDown,
+  faArrowCircleUp,
+  faBolt,
+  faCircle,
+  faCode,
+  faEnvelope,
+  faExclamationCircle,
+  faFile,
+  faFlag,
+  faGift,
+  faRuler,
+  faRulerCombined,
+  faTrash,
+  faTruck
+} from '@fortawesome/free-solid-svg-icons';
+
+// icons used with List by passing string as prop iconClass
+const listIcons = {
+  faArrowCircleDown: faArrowCircleDown,
+  faArrowCircleUp: faArrowCircleUp,
+  faBolt: faBolt,
+  faCircle: faCircle,
+  faCode: faCode,
+  faEnvelope: faEnvelope,
+  faExclamationCircle: faExclamationCircle,
+  faFile: faFile,
+  faFlag: faFlag,
+  faGift: faGift,
+  faRuler: faRuler,
+  faRulerCombined: faRulerCombined,
+  faTrash: faTrash,
+  faTruck: faTruck
+};
```

Comment:
Not necessary but, I find simplified ES6 object properties a little less verbose and easier to read.

```suggestion
const listIcons = {
  faArrowCircleDown,
  faArrowCircleUp,
  faBolt,
  faCircle,
  faCode,
  faEnvelope,
  faExclamationCircle,
  faFile,
  faFlag,
  faGift,
  faRuler,
  faRulerCombined,
  faTrash,
  faTruck
};
```